docs/architecture/adr-007-universal-share-prefix.md

```diff
@@ -0,0 +1,148 @@
+# ADR 007: Universal Share Prefix
+
+## Terminology
+
+- **compact share**: a type of share that can accomodate multiple units. Currently, compact shares are used for transactions, ISRs, and evidence to efficiently pack this information into as few shares as possible.
```

Comment:
```suggestion
- **compact share**: a type of share that can accommodate multiple units. Currently, compact shares are used for transactions, and evidence to efficiently pack this information into as few shares as possible.
```
Typo and I think we got rid of ISRs?

Reply:
The current implementation doesn't contain intermediate state roots. 

https://github.com/rootulp/celestia-app/blob/375093397458a32316a41666264fd2e9d6111219/pkg/appconsts/appconsts.go#L85-L88

Do we expect to include intermediate state roots in the future?